computation_graph/composers/composers_test.py

```diff
@@ -0,0 +1,18 @@
+import time
+
+from computation_graph import composers, graph, run
+
+
+def test_ambiguity_does_not_blow_up():
+    start_time = time.time()
```

Comment:
this is a bit sensitive to the environment.
can we have something that's more deterministic ? like counting the amount of times we applied nodes (not sure it's wasy)?

Reply:
I don't think this is sensitive... unless you have a quantum computer it will be x2000 slower.

Reply:
implemented without `time`